### PR TITLE
[9.0][website_container_fluid] Migrate

### PR DESCRIPTION
**Pull request by yajo**
I removed the workaround hoping that https://github.com/OCA/website/pull/58#issuecomment-131109826 does not happen in v9. Let's see in runbot when built.

I also updated to the new snippet options API.

@Tecnativa.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Tried on runbot. You can consider squash your 8.0 commits if they are meaningless.

File: website_container_fluid/__openerp__.py [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
     "name": "Snippet container width type chooser",
     "summary": "Let you choose between fixed or fluid containers",
-    "version": "8.0.1.0.1",
+    "version": "9.0.1.0.0",
     "category": "Website",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
 
-    "website": "https://odoo-community.org/project/website-maintainers-79",
-    "author": "Grupo ESOC, Odoo Community Association (OCA)",
```
Comment (rafaelbn) [inline website_container_fluid/__openerp__.py]: Please add Tecnativa

Comment (rafaelbn) [inline website_container_fluid/__openerp__.py]: Forget.... it! sorry

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Pull request merged**